plugins/jobs/src/js/components/form/helpers/JobParsers.ts

```diff
@@ -68,7 +68,25 @@ export function jobSpecToOutputParser(jobSpec: JobSpec): JobOutput {
         }
       }
     }
+
+    // RUN CONFIG
+    const { artifacts } = jobSpecCopy.job.run;
+    jobSpecCopy.job.run.artifacts = Array.isArray(artifacts)
+      ? artifacts.filter(_ => _.uri || _.extract || _.executable || _.cache)
```

Comment:
Could we change this from `_` to anything else? I don't know if it's something I picked up or a best practice, but I understand `_` as "omited", like in `function(_, secondParamThatIsTheOnlyOneINeed)`